src/lib/routes/admin-api/favorites.ts

```diff
@@ -75,6 +87,9 @@ export default class FavoritesController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Features'],
+                    summary: 'Remove project from favorites',
+                    description:
+                        'This endpoint removes the project in the url from favorites',
                     operationId: 'removeFavoriteProject',
                     responses: { 200: emptyResponse },
```

Comment:
You know what I'm gonna say by now 💁🏼 
```suggestion
                    responses: { 200: emptyResponse, ...getStandardResponses(401, 404) },
```